specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2021-09-01-preview/appplatform.json

```diff
@@ -3715,6 +3927,161 @@
+        }
+      ],
+      "properties": {
+        "accountName": {
+          "description": "The account name of the Azure Storage Account.",
+          "type": "string",
+        },
+        "accountKey": {
+          "description": "The account key of the Azure Storage Account.",
+          "type": "string",
+        }
+      },
+      "x-ms-discriminator-value": "StorageAccount"
+    },
+    "StorageResourceCollection": {
+      "description": "Collection compose of storage resources list and a possible link for next page.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The storage resources list.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StorageResource"
+          }
+        },
+        "nextLink": {
+          "description": "The link to next page of storage list.",
+          "type": "string"
+        }
+      }
+    },
+    "PersistentVolumeResource": {
+      "description": "Persistent volume resource payload.",
+      "type": "object",
+      "required": [
+        "storageId"
+      ],
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/PersistentVolumeProperties",
+          "description": "Properties of the persistent volume resource payload.",
+          "x-ms-client-flatten": false
+        },
+        "storageId": {
+            "description": "The resource id of storage resource.",
+            "type": "string"
+        }
+      }
+    },
+    "PersistentVolumeProperties": {
+      "description": "Persistent volume resource payload.",
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "The type of the underlying resource to mount as a persistent volume.",
+          "type": "string"
+        },
+        "mountPath": {
+          "description": "The mount path of the persistent volume.",
+          "type": "string"
+        },
+        "mountOptions": {
+          "description": "These are the mount options for a persistent volume.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      },
+      "discriminator": "type"
+    },
+    "PersistentVolumeCollection": {
+      "description": "Collection of persistent volume resources list and a possible link for next page.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The persistent volume resources list.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PersistentVolumeResource"
+          }
+        },
+        "nextLink": {
+          "description": "The link to next page of storage list.",
+          "type": "string"
+        }
+      }
+    },
+    "AzureFile": {
```

Comment:
Would you check whether there is any practice for naming such properties, e.g. is it better to call "PersistentVolumeProperties_AzureFile" or sth like this, in order to avoid conflict if in future we have another thing called "XXX_AzureFile".

Reply:
Changed to AzureFileVolume